smoke/IotEdgeQuickstart/details/IotedgedWindows.cs

```diff
@@ -129,7 +131,8 @@ public async Task Configure(string connectionString, string image, string hostna
 
                 // note: ignore hostname for now
 
-                await Process.RunAsync("powershell", args, cts.Token);
+                string[] result = await Process.RunAsync("powershell", args, cts.Token);
+                WriteToConsole("Output from Configure iotedge windows service", result);
```

Comment:
Again, did you mean to check in this debug output? We don't want to add a lot of clutter to the test output. Ideally, if the test passes there should be very little output.

Reply:
Understand your concern.  This is added for first-time set up environment to run IoTEdgeQuickStart test for troubleshooting.